src/gcp/devConnect.ts

```diff
@@ -3,7 +3,14 @@ import { developerConnectOrigin, developerConnectP4SAOrigin } from "../api";
 import { generateServiceIdentity } from "./serviceusage";
 
 const PAGE_SIZE_MAX = 1000;
-const LOCATION_OVERRIDE = "us-central1"; // TODO(mathusan): for debugging only
+
+/**
+ * LOCATION_OVERRIDE is used to test Developer Connect in different environments.
+ *
+ * Autopush -> 'us-central1'
+ * Staging -> 'us-west1'
```

Comment:
I wonder if we should include this information, I don't see any other references to autopush/staging envs in this repo

Reply:
Also are we planning on removing this after we launch? How is it different from calling `backends:create --location`?

Reply:
> I wonder if we should include this information, I don't see any other references to autopush/staging envs in this repo
ya I think it might cause some confusion. I'll remove. 

> Also are we planning on removing this after we launch? How is it different from calling backends:create --location?

Yes this will be removed as soon as developer connect production is good to go. The reason for this is because we can only create backends in `us-central1` at the moment, and if we were to test in dev connect in staging we need to use `us-west1` because dev connect staging is only spun up in that location.